themes/custom/droopler_theme/js/global.js

```diff
@@ -72,4 +72,14 @@
     }
   };
 
+  /**
+   * Add extra class to first paragraph in section.
+   * @type {{attach: Drupal.behaviors.droopler_first_paragraph.attach}}
+   */
+  Drupal.behaviors.droopler_first_paragraph = {
+    attach: function (context, settings) {
+      $('.paragraph-sections').find('section').first().addClass('first-paragraph');
+    }
+  };
+
```

Comment:
To raczej niepotrzebne. Coś podobnego jest już robione w `_d_p_text_paged.scss`

```
// If this is the first paragraph - decrease top margin.
.paragraph-sections > section:first-child > .paragraph--type--d-p-text-paged {
  margin-top: $d-p-text-paged-first-margin-top;
}
```

Mozna to tylko trochę poprawić, tak aby uwzględniać divy od geysira :thinking: